gno.land/cmd/gnoland/testdata/event.txtar

```diff
@@ -0,0 +1,60 @@
+# start a new node
+gnoland start
+
+gnokey maketx addpkg -pkgdir $WORK -pkgpath gno.land/r/demo/ee -gas-fee 1000000ugnot -gas-wanted 2500000 -broadcast -chainid=tendermint_test test1
+stdout OK!
+stdout 'GAS WANTED: 2500000'
+stdout 'GAS USED: [0-9]+'
+stdout 'EVENTS: null'
```

Comment:
```suggestion
stdout 'EVENTS: []'
```

Reply:
Modified to return a byte slice representing an empty slice where there no events in the `EncodeEvents`.

```go
func (r ResponseBase) EncodeEvents() []byte {
	if len(r.Events) == 0 {
		return []byte("[]")
	}
	res, err := json.Marshal(r.Events)
	if err != nil {
		return nil
	}
	return res
}
```